configure.ac

```diff
@@ -778,6 +778,41 @@ AC_CHECK_LIB(pthread, posix_fallocate,
 	     AC_DEFINE([HAVE_POSIX_FALLOCATE], [1], [The posix library implements file allocation functions.]),
 	     AC_MSG_WARN(POSIX implementation does not have file allocation functions.))
 
+# Check for SCOTCH.
+have_scotch="no"
+AC_ARG_WITH([scotch],
+    [AS_HELP_STRING([--with-scotch=PATH],
+       [root directory where SCOTCH is installed @<:@yes/no@:>@]
+    )],
+    [with_scotch="$withval"],
+    [with_scotch="no"]
+)
+
+SCOTCH_LIBS=""
+if test "x$with_scotch" != "xno"; then
+
+# Check if we have SCOTCH.
+   if test "x$with_scotch" != "xyes" -a "x$with_scotch" != "x"; then
+      SCOTCH_LIBS="-L$with_scotch/lib -lscotch -lscotcherr"
+      SCOTCH_INCS="-I$with_scotch/include"
+   else
+      SCOTCH_LIBS="-lscotch"
```

Comment:
There is a `-lscotcherr` missing here. So should be `SCOTCH_LIBS="-lscotch -lscotcherr"`. With that change
there is no need to add the path on COSMA, a simple `--with-scotch` will do once the module is loaded.